bindings/apple/README.md

```diff
@@ -38,15 +38,19 @@ The `build_crypto_xcframework.sh` script will go through all the steps required
 4. `xcodebuild` an `xcframework` from the fat static libs and the original iOS one, and add the header and module map to it under `generated/MatrixSDKCryptoFFI.xcframework`
 5. cleanup and delete the generated files except the .xcframework and the swift sources (that aren't part of the framework)
 
-## Running the Xcode project
+## Building & testing the Swift package
 
-The Xcode project is meant to provide a simple example on how to integrate everything together but also a place to run unit and integration tests from.
+`Package.swift` is meant to provide a simple example on how to integrate everything together but also a place to run unit and integration tests from.
 
-It's pre-configured to link to the generated .xcframework and .swift files so successfully running the script first is necessary for it to compile.
+It's pre-configured to link to the generated .xcframework and .swift files so successfully running one of the scripts first is necessary for it to compile.
 
-It makes the compiled code available to swift by importing the C header through its bridging header.
+If you're using `debug_build_xcframework.sh`, you can use `xcodebuild` to execute the unit tests in an iOS simulator, e.g.
 
-Once all the generated components are available running it should be as easy as choosing a platform and clicking run.
+```
+xcodebuild test -scheme MatrixRustSDK -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 13'
+```
```

Comment:
So this won't be an option for running the tests anymore. The package is only configured to run on the current platform. So its `debug_build_library.sh` and `swift test` or nothing.

Reply:
Ah so you're planning to entirely remove the `debug_build_xcframework.sh` script with your PR?

Reply:
Not quite. The `debug_build_xcframework.sh` will remain as we still need that for consuming debug builds on EX. The `debug_build_library.sh` script is purely for the tests Swift package.

Reply:
Although, in the future if we hook the static lib build into Xcode's build process, we might be able to get rid of debug XCFrameworks and just use them for distribution.

Reply:
Sorry, I was lacking coffee this morning. 🤦‍♂️ Understood now that you were referring to the removal the iOS platform from `Package.swift`.

I've removed mentions of `debug_build_xcframework.sh` and the command to run the tests in the simulator now.